src/Network/Receive/jRO.pm

```diff
@@ -20,13 +20,17 @@ sub new {
 	my ($class) = @_;
 	my $self = $class->SUPER::new(@_);
 	my %packets = (
+        '009D' => ['item_exists', 'a4 V C v3 C2', [qw(ID nameID identified x y amount subx suby)]],
+        '009E' => ['item_appeared', 'a4 V C v2 C2 v', [qw(ID nameID identified x y subx suby amount)]],
 	);
 
 	foreach my $switch (keys %packets) {
 		$self->{packet_list}{$switch} = $packets{$switch};
 	}
 
 	my %handlers = qw(
+		account_server_info 0B07
+        item_appeared 009E
```

Comment:
it seems to me that this is an extra line

Reply:
Fixed.

Reply:
I don't see any changes, this line is still present in the code

Reply:
This view didn't get updated probably because I did amend and rebase. You can see the "outdated" tag next to the file path. "Files Changed" shows the correct changes.